load-pending-changes.sh

```diff
@@ -8,19 +8,22 @@ set -e
 HOST=localhost
 USER=openchpl_dev
 DUMMY=true
+DATABASE=openchpl
 
 function helpFunction {
     echo ""
     echo "Connection options:"
     echo -e "\t-d do not load dummy data.  Default: [load dummy data]"
     echo -e "\t-h database server host or socket directory.  Default: [localhost]"
     echo -e "\t-u database user name.  Default: [openchpl_dev]"
+    echo -e "\t-b database name.  Default: [openchpl]"
     exit 1 # Exit script after printing help
 }
 
-while getopts 'dh:u:?' flag; do
+while getopts 'd:b:h:u:?' flag; do
```

Comment:
The `d:` part of this means that bash will expect a `-d` flag to have a value after it. Our `-d` command shouldn't have the `:` there, because we use the `-d` as a single flag

```suggestion
while getopts 'db:h:u:?' flag; do
```